db/migrate/20221124173005_add_user_ref_to_post.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPost < ActiveRecord::Migration[7.0]
```

Comment:
There are some migration errors when I try to run `rails db:migrate`. Kindly make sure that there is no database error.

Reply:
@SajjadAhmad14 Hi. If linters were failing you were supposed to invalidate the request weren't you?